functions/image-labelling-node/README.md

```diff
@@ -0,0 +1,85 @@
+# Image labelling example with TensorFlow.js
+
+## Requirements
+
+This example assumes that you are familiar with: 
+ * how serverless functions work. If needed, you can check [Scaleway official documentation](https://www.scaleway.com/en/docs/serverless/functions/quickstart/).
+ * how S3 object storage works, and especially how to create a bucket and upload files within a bucket. Please refer to scaleway's documentation [here](https://www.scaleway.com/en/docs/storage/object/quickstart/).
+ * how to create and get user API access and secret keys using IAM. Please refer to IAM documentation [here](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/).
+
+This example uses the Scaleway Serverless Framework Plugin. Please set up your environment with the requirements stated in the [Scaleway Serverless Framework Plugin](https://github.com/scaleway/serverless-scaleway-functions) before trying out the example.
+
+
+## Context
+
+This example shows how to label an RGB image in an S3 bucket using serverless functions. The example uses a pre-trained ready-to-use model from [TensorFlow.js](https://www.tensorflow.org/js/models). The model is called `mobilenet` and can be used on server or client side. It returns three labels of an image with their respective prediction probabilities (namely, logits). Check `mobilenet` Github repository [here](https://github.com/tensorflow/tfjs-models/tree/master/mobilenet).
+
+
+## Description
+
+The function gets an RGB image (jpg, jpeg, or png formats) from an S3 bucket, transforms it into a TensorFlow-compatible object (namely, a tensor), and then applys a pretrained `mobilenet` model to return the labels and their respective logits in a json format. 
+
+This function uses Node 18 runtime. Used package are specified in `package.json`.
+
+## Setup
+
+### Upload an image on an S3 bucket 
+
+Create an S3 bucket and upload an RGB image (jpg, jpeg or png format) within this bucket. Keep the name of your bucket and the name of your file (namely, the source key). 
+
+### Fill environment variables
+
+Fill your environment variables within `serverless.yml` file:
+
+```
+secret:
+    USER_ACCESS_KEY: <bucket scw access key>
+    USER_SECRET_KEY: <bucket scw access key id>
+    S3_ENDPOINT_URL: http://s3.fr-par.scw.cloud
+```
```

Comment:
Alternatively people can just set up the Scaleway CLI, and Serverless framework will automatically read the credentials from their `~/.config/scw/config.yaml` file.